workspaces/liferay-sample-workspace/client-extensions/liferay-sample-theme-spritemap-1/client-extension.yaml

```diff
@@ -1,7 +1,8 @@
 assemble:
     - from: assets
+      hashify: spritemap.svg
```

Comment:
is this some sort of black magig? wasn't aware of it. For single files not imported elsewhere, this is a nice option!

Reply:
It was introduced in the favicon PR when I suggested to create a task to hashify files. Brian made it with Peter Shin.